### PR TITLE
fix(inpage-nav): fix mobile issue - INNO-943

### PR DESCRIPTION
**Pull request by yhuard**
Fix the inpage navigation issue reported in https://github.com/ec-europa/europa-component-library/issues/593. It was basically a structural issue. 

I also "updated" the generic component and its systems-related counterparts.

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit b5337ee9b7e8109f5bbb8870b6b292275d05c86c

https://deploy-preview-613--europa-component-library.netlify.com

File: src/systems/ec/ec-component/ec-component-navigation/ec-component-inpage-navigation/ec-component-inpage-navigation.twig [hunk 1]
```diff
@@ -7,41 +7,4 @@
...
-          'extra_classes': 'ecl-inpage-navigation__link',
-          'variant': 'standalone',
-          }|merge(link|default({}))
-        %}
-      </li>
-      {% endfor %}
-    </ul>
-  </div>
-</nav>
+{% include '@ecl/generic-component-inpage-navigation' with { system: 'ec' } %}
```
Comment (emeryro) [inline src/systems/ec/ec-component/ec-component-navigation/ec-component-inpage-navigation/ec-component-inpage-navigation.twig]: I understand the need to pass the system to parent template, but this is somehow redundant (we are using an EC component, so we should use system: ec in any cases).
Maybe we could find a way to pass this parameter automatically (could be done in another task, not a blocker)

**Review by emeryro — COMMENTED** [review]
changes are working fine.
One remark: inpage navigation is only working when displaying the component on full screen (not in the demo). But I guess this is acceptable. Maybe we should just add a line about it in the readme to prevent confusion.

Also see comment below

**Comment by yhuard** [comment]
Interesting, I didn't test it in the "demo" iframe. That's weird

File: src/systems/ec/ec-component/ec-component-navigation/ec-component-inpage-navigation/ec-component-inpage-navigation.twig (hunk 1 above)
Comment (yhuard) [reply src/systems/ec/ec-component/ec-component-navigation/ec-component-inpage-navigation/ec-component-inpage-navigation.twig]: This was the simplest solution I came up to. If you have a better idea, I'll welcome it :)

Comment (emeryro) [reply src/systems/ec/ec-component/ec-component-navigation/ec-component-inpage-navigation/ec-component-inpage-navigation.twig]: not yet, I'll open a ticket to think about it

**Comment by yhuard** [comment]
It seems the class "ecl-inpage-navigation--sticky" is not applied in the demo

**Review by emeryro — APPROVED** [review]
review ok

**Pull request merged**